libfwupdplugin/fu-udev-device.h

```diff
@@ -75,6 +125,10 @@ void
 fu_udev_device_set_bind_id(FuUdevDevice *self, const gchar *bind_id);
 const gchar *
 fu_udev_device_get_driver(FuUdevDevice *self);
+gboolean
+fu_udev_device_is_pci_base_class(FuUdevDevice *self, FuPciBaseClass cls);
+guint32
```

Comment:
guint8, or FuPciBaseClass perhaps?

Reply:
It's actually intentional that it returns the whole class, so if we have a need later we can add subclasses

Reply:
sure